libbeat/processors/fingerprint/hash.go

```diff
@@ -51,3 +51,7 @@ func (f *hashMethod) Unpack(str string) error {
 	*f = m
 	return nil
 }
+
+func newXxHash() hash.Hash {
```

Comment:
Could you add a comment here as to why this wrapper function is necessary, especially since one isn't necessary for the other hash methods? I just put up https://github.com/cespare/xxhash/pull/36; if/when that is merged and released, we won't need this wrapper function here.

Reply:
Looks like cespare/xxhash#36 won't be merged. So no point mentioning it in a comment, but I think it would still be good to have a comment here about the need for the wrapper.